### PR TITLE
WEB-598: HeroCard

### PR DESCRIPTION
**Pull request by oddwili**
[WEB-598](https://bostoncommonpress.atlassian.net/browse/WEB-598)

HeroCard without desktop styling finalized. I left in some TODOs to clean up later.

### CODE REVIEW

File: v2/src/components/blocks/HeroCard/HeroCard.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      <div className={styles.info}>
+        {image && (
+          <img
+            className={styles.heroImage}
+            alt={image?.altText}
+            src={image?.cloudinaryUrl}
+          />
+        )}
+        <div className={styles.text}>
+          <h1 className={styles.headline}>Your Summer Needs These Three Tomato Sandwiches</h1>
```
Comment (tdorin) [inline v2/src/components/blocks/HeroCard/HeroCard.tsx:26]: Thinking that this h1 headline should probably be included in your props vs being hardcoded.

File: v2/src/components/blocks/HeroCard/styles/HeroCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
+@use "./src/styles/mise.scss";
+
+.container {
+    @include mise.font(display, primary, sm);
+
+    color: mise.color(neutral, white, base);
+    height: 548px;
+    background-image: url('https://res.cloudinary.com/hksqkdlah/image/upload/SFS_Broiled_Chicken_with_Gravy_379_rx4dzb.jpg');
```
Comment (tdorin) [inline v2/src/components/blocks/HeroCard/styles/HeroCard.module.scss:8]: isn't the image passed as a prop?

File: v2/src/components/blocks/HeroCard/HeroCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  image:
+  {
+    altText: 'Kevin Pang',
+    cloudinaryUrl: 'https://res.cloudinary.com/hksqkdlah/image/upload/c_fill,dpr_2.0,f_auto,fl_lossy.progressive.strip_profile,g_faces:auto,q_auto:low/AKO%20Articles/Author_Headshots/staff_kevin_pang'
+  },
+  inactive: false,
+}
+
+export const HeroCardStory: ComponentStory<typeof HeroCard> = Template.bind({});
+HeroCardStory.args = {
```
Comment (tdorin) [inline v2/src/components/blocks/HeroCard/HeroCard.stories.tsx:26]: Should probably include for a headline too.

**Review by tdorin — CHANGES_REQUESTED** [review]
Just a few questions about props.

**Comment by bradwaropay** [comment]
Agreed to all @tdorin comments. I would make those props and just set a default.

File: v2/src/components/blocks/HeroCard/styles/HeroCard.module.scss (hunk 2 above)
Comment (oddwili) [reply v2/src/components/blocks/HeroCard/styles/HeroCard.module.scss:8]: oop, yes!

**Pull request merged**